src/js/vuex-store/report.js

```diff
@@ -163,5 +168,8 @@ export default {
     async loadData({ dispatch }) {
       return Promise.all([dispatch('loadAreaData'), dispatch('loadCountyAverages')]);
     },
+    setActiveCategory({ commit }, category) {
```

Comment:
I don't think this doesn't need to be an action, it can be just used as a mutation directly in the components; unless there's some reason to make it an action? Actually, ideally we could just store this in the Report component state rather than using Vuex at all but I think this approach is totally fine and there are other places where I'm also over-using Vuex already lol!